docs/resources/account_network_policy.md

```diff
@@ -0,0 +1,58 @@
+---
+subcategory: "Settings"
+---
+# databricks_account_network_policy Resource
+
```

Comment:
We need to have a note that says in which context this resource or data source could be used - account or workspace...

Reply:
Discussed offline, summary: This would be added properly with description and example usage.